pycmr/queries.py

```diff
@@ -198,10 +289,21 @@ def query(self):
         options_as_string = "&".join(formatted_options)
 
         url = "{}?{}&{}".format(self.base_url, params_as_string, options_as_string)
-
+        print(url)
         response = get(url)
         return response.json()
```

Comment:
Print just for now / debugging? otherwise I guess we could just do `return get(url).json()`